doc/source/architecture.rst

```diff
@@ -150,7 +151,7 @@ Typical Scenario
 
 It is more typical to have several MySQL and MongoDB server instances
 distributed over different hosts.
-In this case, you would run *PMM Server* on a dedicated monitoring host,
+In this case, you can run *PMM Server* on a dedicated monitoring host,
 and install *PMM Client* on every database host that you want to monitor.
 Data from hosts will be aggregated on the PMM Server.
 
```

Comment:
mysqld_exporter is now custom https://github.com/percona/mysqld_exporter based on the official one github.com/prometheus/mysqld_exporter